platforms/xilinx_vck5000_air/vivado/xilinx_vck5000_air_xsa.tcl

```diff
@@ -3,7 +3,7 @@
 # SPDX-License-Identifier: MIT
 
 proc numberOfCPUs {} {
-    return 8
+    return 16
```

Comment:
Perhaps a comment in the code would help understanding the relation to the PR subject.

Reply:
This is just the number of threads used by Vivado